lib/sqs/models/QueueConfig.rb

```diff
@@ -50,11 +50,11 @@ def to_hash
       #
       # attributes - the queue attributes in AWS
       def populate!(attributes)
-        @delay = if attributes["DelaySeconds"] then attributes["DelaySeconds"].to_i end
-        @max_message_size = if attributes["MaximumMessageSize"] then attributes["MaximumMessageSize"].to_i end
-        @message_retention = if attributes["MessageRetentionPeriod"] then attributes["MessageRetentionPeriod"].to_i end
-        @receive_wait_time = if attributes["ReceiveMessageWaitTimeSeconds"] then attributes["ReceiveMessageWaitTimeSeconds"].to_i end
-        @visibility_timeout = if attributes["VisibilityTimeout"] then attributes["VisibilityTimeout"].to_i end
+        @delay = if attributes["DelaySeconds"] then attributes["DelaySeconds"] end
```

Comment:
although it really doesn't matter if the .to_i is there, I changed it to match the other models now that the json isn't a string.